flatsurvey/jobs/undetermined_interval_exchange_transformation.py

```diff
@@ -110,9 +111,28 @@ def __init__(
         flow_decompositions,
         saddle_connection_orientations,
         cache,
+        cache_only=Goal.DEFAULT_CACHE_ONLY,
         limit=DEFAULT_LIMIT,
     ):
-        super().__init__(producers=[flow_decompositions])
+        super().__init__(producers=[flow_decompositions], cache=cache, cache_only=cache_only)
+
+    async def consume_cache(self):
```

Comment:
For some of the goals, `consume_cache` should probably just be a noop. In such cases, there should also be no `--cache-only` switch in the CLI.